docs/_static/cookie_notice.css

```diff
@@ -0,0 +1,33 @@
+#cookie-notice {
+  display: none;
+  bottom: 0;
+  position: fixed;
+  width: 100%;
+  background:white;
+  z-index:1000000;
+  height: auto;
+  text-align: center;
+}
+.cookie-button {
+  border: none;
+  display: inline-block;
+  justify-content: center;
+  height: 48px;
+  font-size: 14px;
+  font-weight: 600;
+  border-radius: 4px;
+  background: radial-gradient(circle, #4C71C6 0%, #002B86 100%);
+  box-shadow: 0 5px 14px 0 rgba(29,30,36,0.19);
+  color: #fff;
+  text-transform: uppercase;
+  letter-spacing: 1.2px;
+  cursor: pointer;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  padding-left: 42px;
+  padding-right: 42px;
+  margin: 1em;
+  margin-left: 2em;
+  white-space: nowrap;
+  vertical-align: middle;
+}
```

Comment:
This is Leap theme banner on RtD theme. It just doesn't feel right. :smile:

Reply:
Btw, to use something closer to RtD, it's enough to change the accept button. A few lines:
```
-  font-size: 14px;
+  font-size: 16px;

-  background: radial-gradient(circle, #4C71C6 0%, #002B86 100%);
+  background: rgb(41, 128, 185);

-  text-transform: uppercase;
-  letter-spacing: 1.2px;
```

Reply:
Will change, thanks